tests/test_hbavss_batch.py

```diff
@@ -0,0 +1,48 @@
+from pytest import mark
+from random import randint
+from contextlib import ExitStack
+from honeybadgermpc.polynomial import polynomials_over
+from honeybadgermpc.betterpairing import G1, ZR
+from honeybadgermpc.hbavss_batch import HbAvssBatch
+from honeybadgermpc.mpc import TaskProgramRunner
+import asyncio
+
+
+def get_avss_params(n, t):
```

Comment:
This should be moved to `fixtures` and reused

Reply:
Files merged. No need for the move.